eemeter/meter/seasonal_meter.py

```diff
@@ -0,0 +1,586 @@
+                        has_enough_periods_with_low_cdd_per_day: {},
+                        meets_cvrmse_limit: {},
+                    },
+                },
+            }
+        },
+        !obj:eemeter.meter.And {
+            inputs: [
+                has_enough_total_hdd,
+                has_enough_periods_with_high_hdd_per_day,
+                has_enough_periods_with_low_hdd_per_day,
+            ],
+            input_mapping: {
+                has_enough_total_hdd: {},
+                has_enough_periods_with_high_hdd_per_day: {},
+                has_enough_periods_with_low_hdd_per_day: {},
+            },
+            output_mapping: { output: { name: has_enough_hdd, }, },
+        },
+        !obj:eemeter.meter.And {
+            inputs: [
+                has_enough_total_cdd,
+                has_enough_periods_with_high_cdd_per_day,
+                has_enough_periods_with_low_cdd_per_day,
+            ],
+            input_mapping: {
+                has_enough_total_cdd: {},
+                has_enough_periods_with_high_cdd_per_day: {},
+                has_enough_periods_with_low_cdd_per_day: {},
+            },
+            output_mapping: { output: { name: has_enough_cdd, }, },
+        },
+        !obj:eemeter.meter.And {
+            inputs: [
+                has_enough_hdd,
+                has_enough_cdd
+            ],
+            input_mapping: {
+                has_enough_hdd: {},
+                has_enough_cdd: {},
+            },
+            output_mapping: { output: { name: has_enough_hdd_cdd }, }
+        },
+        !obj:eemeter.meter.And {
+            inputs: [
+                spans_184_days,
+                has_enough_hdd_cdd
+            ],
+            input_mapping: {
+                spans_184_days: {},
+                has_enough_hdd_cdd: {},
+            },
+            output_mapping: { output: { name: spans_183_days_and_has_enough_hdd_cdd, }, }
+        },
+        !obj:eemeter.meter.Or {
+            inputs: [
+                spans_330_days,
+                spans_183_days_and_has_enough_hdd_cdd
+            ],
+            input_mapping: {
+                spans_330_days: {},
+                spans_183_days_and_has_enough_hdd_cdd: {},
+            },
+            output_mapping: { output: { name: has_enough_data } }
+        },
+        !obj:eemeter.meter.And {
+            inputs: [
+                has_recent_reading,
+                has_enough_data,
+                meets_cvrmse_limit,
+            ],
+            input_mapping: {
+                has_recent_reading: {},
+                has_enough_data: {},
+                meets_cvrmse_limit: {},
+            },
+            output_mapping: { output: { name: meets_model_calibration_utility_bill_criteria }, }
+        }
+    ]
+}
+"""
+
+class SeasonalModelMeter(YamlDefinedMeter):
+    """Based on the construction of BPI_2400_S_2012_ModelCalibrationUtilityBillCriteria.
```

Comment:
Is there a particular reason you decided to do this rather than use the `BPI_2400_S_2012_ModelCalibrationUtilityBillCriteria` class directly? Just wondering if it wasn't general enough.